common/changes/@uifabric/example-app-base/applied-prettier_2018-06-02-22-17.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
```

Comment:
This file should not be showing up as a new file created by you..

Reply:
I pulled upstream/master into my local branch, committed, and then pushed to origin. I'd definitely appreciate tips for how to avoid having these files show up in my PRs.